index.html

```diff
@@ -1,206 +1,405 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <title> David Portfolio </title>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <title> David Portfolio </title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" media="(min-width:451px)" href="tablet.css">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
-<body> 
-  <div class="">
 
-  <div class ="stage_1">
-    <header class>
-      <a href="#"class="logo">My Logo</a>
+<body>
+  <div class="section-1">
+    <header>
+      <div class="logo">
+        <a href="#" class="logo">My Logo</a>
+      </div>
       <nav>
-        <a href="javascript:void(0);" class="icon">
-          <i class="fa fa-bars"></i>
-        </a>
+        <div class="icon-div desktop-hidden">
+          <a href="javascript:void(0);" class="icon desktop-hidden">
+            <i class="fa fa-bars"></i>
+          </a>
+        </div>
+        <div class="nav-links mobile-hidden">
+          <ul>
+            <li><a href="#" class="nav-link">Portfolio</a></li>
+            <li><a href="#" class="nav-link">About</a></li>
+            <li><a href="#" class="nav-link">Contact</a></li>
+          </ul>
+        </div>
       </nav>
     </header>
 
-    <section class = "section-1">
-      <h1> I'm David <br>Glad to see <br>you!</h1>
-      <p> I’m a software developer! I can help you build a product,
-        feature or website Look through some of my
-        work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-      </p>
-      <a href="#"> LET'S CONNECT</a>
+    <section class="bio-section">
+      <div class="head-1">
+        <h1> I'm David <br>Glad to see you!</h1>
+        <p> I’m a software developer! I can help you build a product,
+          feature or website Look through some of my
+          work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+        <a href="#"> LET'S CONNECT</a>
+      </div>
```

Comment:
- [OPTIONAL] It is required to stick to the Figma design as close as possible. This part of your design on the desktop version does not look exact like the figma design. I would recommend you to give it a time to modify it. So that it will make your portfolio looks more standard.

### Your Design
![your-headline](https://user-images.githubusercontent.com/26597432/215281787-a06b29a4-175d-4a01-b498-bc4563dc7702.png)

### Figma Design
![temp-headline](https://user-images.githubusercontent.com/26597432/215281798-56f28a7f-a0d4-479e-bbbf-dea00697c929.png)